dotcom-rendering/src/web/components/NavList.tsx

```diff
@@ -28,7 +28,10 @@ const ulStyles = css`
 	}
 	column-count: 4;
 
-	column-rule: 1px solid ${border.secondary};
+	column-rule: 1px solid
+		${containerOverrides
+			? containerOverrides.border.container
+			: border.secondary};
```

Comment:
What follows is information about Emotion and does not have to be implemented as part of this PR.

In order to keep the dynamic and static parts of the CSS distinct, which reduces the runtime work Emotion has to do, you could extract the [`column-rule-color`](https://developer.mozilla.org/en-US/docs/Web/CSS/column-rule-color) to its own style declaration, like so:

```ts
columnRuleColor = (containerOverrides?: ContainerOverrides) => css`
	column-rule-color: ${containerOverrides
		? containerOverrides.border.container
		: border.secondary};
`;

/* later… */
<ul css={[ulStyles, columnRuleColor(containerOverrides)}>
```